src/components/PathwaysModal/PathwaysModal.tsx

```diff
@@ -0,0 +1,466 @@
+            <button className="relative inline-flex items-center justify-center p-0.5 mb-2 me-2 overflow-hidden text-sm font-medium text-gray-900 rounded-lg group bg-gradient-to-br from-green-400 to-blue-600 group-hover:from-green-400 group-hover:to-blue-600 hover:text-white dark:text-white focus:ring-4 focus:outline-none focus:ring-green-200 dark:focus:ring-green-800">
+              <span className="relative px-5 py-2.5 transition-all ease-in duration-75 bg-white dark:bg-gray-900 rounded-md group-hover:bg-opacity-0">
+                Free Data NFTs!
+              </span>
+            </button>
+          </a>
+
+          <div className="text-sm font-medium text-gray-900 dark:text-white">
+            <a
+              href="https://datadex.itheum.io/datanfts/marketplace/DATANFTFT-e936d4-07/offer-327"
+              target="_blank"
+              className="block w-full px-4 py-2 border-b border-gray-200 cursor-pointer hover:bg-gray-100 hover:text-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-700 focus:text-blue-700 dark:border-gray-600 dark:hover:bg-gray-600 dark:hover:text-white dark:focus:ring-gray-500 dark:focus:text-white">
+              BiTz XP
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function DataNFTSolana() {
+  return (
+    <div className="bg-red-0 grid grid-cols-4 gap-4 mb-4 items-start w-[100%]">
+      <div className="flex items-center justify-center rounded bg-gray-50 dark:bg-gray-800">
+        <div className="text-2xl text-gray-400 dark:text-gray-500">
+          <a href="https://docs.google.com/forms/d/1BbpcZiFaE30ctZJLzBP356RlRfplF-ElnNXSQos0Fuw/edit" target="_blank">
+            <button className="relative inline-flex items-center justify-center p-0.5 mb-2 me-2 overflow-hidden text-sm font-medium text-gray-900 rounded-lg group bg-gradient-to-br from-green-400 to-blue-600 group-hover:from-green-400 group-hover:to-blue-600 hover:text-white dark:text-white focus:ring-4 focus:outline-none focus:ring-green-200 dark:focus:ring-green-800">
+              <span className="relative px-5 py-2.5 transition-all ease-in duration-75 bg-white dark:bg-gray-900 rounded-md group-hover:bg-opacity-0">
+                Launch Data NFTs
+              </span>
+            </button>
+          </a>
+
+          <div className="text-sm font-medium text-gray-900 dark:text-white">
+            <a
+              href="https://docs.google.com/forms/d/1BbpcZiFaE30ctZJLzBP356RlRfplF-ElnNXSQos0Fuw/edit"
+              target="_blank"
+              aria-current="true"
+              className="block w-full px-4 py-2 border-b border-gray-200 cursor-pointer hover:bg-gray-100 hover:text-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-700 focus:text-blue-700 dark:border-gray-600 dark:hover:bg-gray-600 dark:hover:text-white dark:focus:ring-gray-500 dark:focus:text-white">
+              Drip Haus
```

Comment:
Shouldn't this be written as DRiP?

Reply:
Pathways is a WIP -- will be fixed in next release